test/utils/logger.test.js

```diff
@@ -6,25 +7,7 @@ const config = require('../../lib/config');
 const { LOG_PREFIX } = require('../../lib/constants');
 const { mockConfig } = require('../test-helpers/mock-config');
 
-const {
```

Comment:
Use of `jest.requireActual` was redistributed throughout the module wherever feasible so mocks can still be used (e.g., for spying on `getLogLevel`)